src/UI.tsx

```diff
@@ -21,7 +20,7 @@ const nothing = () => {
 /**
  * Removes scrollbar from the page and contain the scroll within the Lock
  */
-const RemoveScroll: RemoveScrollUIType = React.forwardRef<HTMLElement, IRemoveScrollUIProps>((props, parentRef) => {
+const RemoveScroll: RemoveScrollUIType = ({ ref: parentRef, ...props }) => {
```

Comment:
this instantly drops support for anything before React 19 and thus required major version bump.
Not something I can do.